### DISCUSSION TITLE
Keycloakify older keycloak version support

### DISCUSSION THREAD

**Discussion by maciejKrasny** [keycloakify/keycloakify#363] [Q&A]
Hello,
I would like to ask if Keycloakify's latest version is compatible with an older version of Keycloak. Currently, in the project, we have v14 of keycloak and we would like to introduce Keycloakify. Is it possible to do this with v14 version or is it necessary to migrate keycloak to the latest one? Keycloak pretty often updates his major version, how about compatibility in the future?

**Comment by garronej** [comment]
Keycloakify is fully compatible with Keycloak, starting from version 11 and is anticipated to maintain compatibility with all future versions. To understand the basis of my confidence in this, you can visit this discussion thread where I've explained in detail: https://github.com/keycloakify/keycloakify/discussions/346.
